src/pineko/scale_variations.py

```diff
@@ -139,10 +143,18 @@ def write_sv_grids(gridpath, grid_list):
     base_name = gridpath.stem.split(".pineappl")[0]
     final_part = ".pineappl.lz4"
     grid_paths = []
-    for num, grid in zip(range(len(grid_list)), grid_list):
-        new_grid_path = gridpath.parent / (base_name + "_" + str(num) + final_part)
+    for order in grid_list:
+        if len(grid_list[order]) > 1:
+            for grid in grid_list[order][1:]:
+                tmp_path = gridpath.parent / ("tmp" + final_part)
+                grid.raw.write_lz4(tmp_path)
+                grid_list[order][0].raw.merge_from_file(tmp_path)
+                os.remove(tmp_path)
```

Comment:
[No need](https://docs.python.org/3/library/pathlib.html?highlight=pathlib#pathlib.Path.unlink) for `os` (as usual :upside_down_face: )

Reply:
Yes, thanks for the comment. Anyway this is just a draft, it is going to change a lot in the near future